backend/src/controllers/reports/genericReportController.js

```diff
@@ -2,6 +2,7 @@
 const useCommonComponents = require("../useCommonComponents/index");
 const useController = require("../useController/index");
 const cdogs = useCommonComponents("cdogs");
+//const _ = require("lodash");
```

Comment:
Does this commented-out code need to be here?

Reply:
no this one has no use